airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/check/DestinationChecker.kt

```diff
@@ -25,7 +25,7 @@ interface DestinationChecker<C : DestinationConfiguration> {
         DestinationStream(
             descriptor = DestinationStream.Descriptor("testing", "test"),
             importType = Append,
-            schema = NullType,
+            schema = ObjectTypeWithoutSchema,
```

Comment:
tangential, but I suspect we'll eventually change this to something like `ObjectType(mapOf("foo" to StringType))` (mostly for warehouse destinations, where creating a table with no columns is probably dumb)